euth/follows/serializers.py

```diff
@@ -8,6 +9,7 @@
 class FollowSerializer(serializers.ModelSerializer):
 
     project = serializers.SlugRelatedField(read_only=True, slug_field='slug')
+    follows = serializers.SerializerMethodField()
```

Comment:
More informal than a change request: There is a similar approach in the [rating api](https://github.com/liqd/euth_wagtail/blob/master/euth/ratings/serializers.py#L7), where also additional information is serialised as `meta_info` dict.